memory_profiler.py

```diff
@@ -128,9 +155,11 @@ def ps_util_tool():
     def posix_tool():
         # .. scary stuff ..
         if include_children:
-            raise NotImplementedError('The psutil module is required when to'
-                                      ' monitor memory usage of children'
-                                      ' processes')
+            raise NotImplementedError((
```

Comment:
Sorry about this, I think this was just a carry over from the historical merge; I can reset back to the original if needed.